minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -45,6 +47,7 @@
 /**
  * Local Tachyon cluster.
  */
+@NotThreadSafe
```

Comment:
What's the convention for annotating interfaces/abstract classes & their implementations?